### DISCUSSION TITLE
VS2022, .Net8, Blazorise App (Megabit) template

### DISCUSSION THREAD

**Discussion by domenicocanavero** [Megabit/Blazorise#5646] [Q&A] [answered]
Create a new project with **Provider (CSS) -> Material**, these are the contents of the wwwroot folder:
![image](https://github.com/user-attachments/assets/623d9ac4-92e3-4099-87f0-01e7a5b54e4c)

Create another new project with Provider (CSS) - Bootstrap5, these are the contents of the wwwroot folder:
![image](https://github.com/user-attachments/assets/c8e5faee-52dc-4b45-bdc0-e7baa97a8e83)

The same contents!

Why?

My mistake?

**Comment by David-Moreira** [comment, marked as answer]
If that's what's being generated, that seems like a bug on our template, we need to take a look at it.
